puterbot/scrub.py

```diff
@@ -0,0 +1,54 @@
+from presidio_analyzer import AnalyzerEngine
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import OperatorConfig
+
+MAX_MASK_LEN = 1024
+
+# Required Packages: 
+# pip install presidio_analyzer
+# pip install presidio_anonymizer
+# pip install presidio_image_redactor
+# Note: they have been added in requirement.txt
+# python -m spacy download en_core_web_lg (after installing the required packages)
+
+def scrub(text: str) -> str:
+    """
+        This function takes in a string of text as input and returns a scrubbed version of the text with any potential personally
```

Comment:
Thank you for the docstring!

Please follow the format described here: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods